File: iFlat/web/app/model/system/DataDictionary.js
Ext.define('iFlat.model.system.DataDictionary', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'dataDictionary.dbInstance', mapping: 'dbInstance', type: 'string'},
        {name: 'dataDictionary.dbName', mapping: 'dbName', type: 'string'},
        {name: 'dataDictionary.tableName', mapping: 'tableName', type: 'string'},
        {name: 'dataDictionary.fieldName', mapping: 'fieldName', type: 'string'},
        {name: 'dataDictionary.alias', mapping: 'alias', type: 'string'},
        {name: 'dataDictionary.type', mapping: 'type', type: 'string'},
        {name: 'dataDictionary.length', mapping: 'length', type: 'int'},
    ]
});

File: src/main/java/com/iflat/system/bean/DataDictionary.java
package com.iflat.system.bean;

/**
 * Created by tyriv on 2015/9/20.
 */
public class DataDictionary {

    //数据库实例
    private String dbInstance;
    //数据库名
    private String dbName;
    //表名
    private String tableName;
    //字段
    private String fieldName;
    //别名
    private String alias;
    //类型
    private String type;
    //长度
    private int length;

    public String getDbInstance() {
        return dbInstance;
    }

    public void setDbInstance(String dbInstance) {
        this.dbInstance = dbInstance;
    }

    public String getDbName() {
        return dbName;
    }

    public void setDbName(String dbName) {
        this.dbName = dbName;
    }

    public String getTableName() {
        return tableName;
    }

    public void setTableName(String tableName) {
        this.tableName = tableName;
    }

    public String getFieldName() {
        return fieldName;
    }

    public void setFieldName(String fieldName) {
        this.fieldName = fieldName;
    }

    public String getAlias() {
        return alias;
    }

    public void setAlias(String alias) {
        this.alias = alias;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public int getLength() {
        return length;
    }

    public void setLength(int length) {
        this.length = length;
    }
}
